docs/05-engineering/accessibility.md

```diff
@@ -4,7 +4,7 @@ We implement 508 and WCAG compliant websites so that people with all types of di
 
 ## How we do this
 
-- We use base themes which implement ADA and WCAG guidelines.
+- We use base themes which stive to meet the W3C's WCAG 2.0 or above. This will help satisfy the requirements of the ADA.
```

Comment:
Typo - should be "strive"?

Reply:
It looks like I've edited it at 534167b but not sure I'm doing this right.  Sorry for the typo.

Reply:
That works!